### DISCUSSION TITLE
Can Allure select to run only a particular set of parameters in a parametrized test?

### DISCUSSION THREAD

**Discussion by erwin-wee** [allure-framework/allure2#1878] [Ideas & Feedback]
I have a parametrized test that has say for example has 100 permutations of 5 parameters. Let's say I run a test for all 100 permutations and 1 of them fail. Is it possible to select "re-run" from Allure Testops and only run that 1 permutation? How can I do that?

Context: I have Allure Testops configured to integrate with CircleCI, running a job that triggers pytest, configured as per documentation. I'm able to selectively choose test cases to run, but so far not successful with getting a particular permutation of parameters to run just a single test instead of 100 for a parametrized test.

**Comment by cheshi-mantu** [comment]
Hi @erwin-wee 
none of adaptors for the tests frameworks allow selection of a particular parameters as of yet. Only certain tests selection is possible.

**Comment by erwin-wee** [comment]
I see, thanks for the reply. I'm changing this to an idea. It'd be great to be able to support this so that for failed tests I can re-run only tests that failed in a particular run and not all parameters in a parameterized test.

**Reply by cheshi-mantu** [reply]
it's better to create a feature request in appropriate repo, @erwin-wee
